src/reducers/src/vis-state-updaters.ts

```diff
@@ -1416,8 +1417,14 @@ export const receiveMapConfigUpdater = (
   // reset config if keepExistingConfig is falsy
   let mergedState = !keepExistingConfig ? resetMapConfigUpdater(state) : state;
   for (const merger of state.mergers) {
-    if (isValidMerger(merger) && config.visState[merger.prop]) {
-      mergedState = merger.merge(mergedState, config.visState[merger.prop], true);
+    // @ts-expect-error 'ParsedVisState' is not assignable to parameter of type 'VisState'
```

Comment:
try change `hasPropsToMerge`'s arguments to be
`export function hasPropsToMerge<State extends {}>(`

and `getPropValueToMerger` to be 
`export function getPropValueToMerger<State extends {}>(`